src/UnsecureBundle/DependencyInjection/Configuration.php

```diff
@@ -20,6 +20,10 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $rootNode = $treeBuilder->root('unsecure');
 
+        $rootNode = array_map(function () {
```

Comment:
What did you add here ?

Reply:
Just an `array_map` function to return random number 👌

Reply:
There is not native function to return random number ?